@xen-orchestra/lite/src/components/ui/modals/ModalApproveButton.vue

```diff
@@ -0,0 +1,13 @@
+<template>
```

Comment:
Update `modal layout stories` to use `ModalApproveButton` and `ModalDeclineButton` insteadn `UiButton`